cfme/tests/infrastructure/test_providers.py

```diff
@@ -189,6 +189,7 @@ def test_providers_discovery(request, provider):
     wait_for_a_provider()
 
 
+@pytest.mark.skipif(lambda provider: provider.type == 'rhevm', 'blocker=1399622')
```

Comment:
It skipped for me with rhevm - can you test this patch with some vmware setup to make sure it doesn't skip for vmware?